content/blog/newsletter/2025-11-05-newsletter.md

```diff
@@ -0,0 +1,233 @@
+- [Un nuovo server di chat per la libertà di comunicare: la nostra risposta a ChatControl](https://estelinux.serviziliberi.it/un-nuovo-server-di-chat-per-la-liberta-di-comunicare-la-nostra-risposta-a-chatcontrol/) by [Fabio Lovato](https://loviuz.it/) for [ILS Este](https://estelinux.serviziliberi.it/). [IT]
+- [End-to-End Encrypted Chat that YOU Control: Hosting XMPP (Jabber) with Prosody](https://roguesecurity.dev/blog/xmpp) by [starkzarn](https://roguesecurity.dev/authors/starkzarn) for the [RogueSecurity blog](https://roguesecurity.dev/blog).
+- [New home for the slidge room(s!)](https://slidge.im/blog/2025/10/19/moving-away-from-nicoco.fr/), from the [Slidge.im blog](https://slidge.im/blog/).
+- [Libervia CLI tips](https://mastodon.social/@Goffi/tagged/tips): [Goffi](https://mastodon.social/@Goffi) keeps adding them and the list keeps growing with more and more [CLI](https://libervia.org/__b/doc/backend/libervia-cli/) tips to make it easier for you to use and do nearly everything you need with [Libervia](https://libervia.org/) from the comfort of your terminal!
+
+## XMPP Software News
+
+### XMPP Clients and Applications
+
+- [Monocles](https://monocles.eu/) has released version [2.0.16](https://codeberg.org/monocles/monocles_chat/releases/tag/CodebergRelease-v2.0.16) of its chat client for Android. This update comes **loaded** with features and improvements; way too many to list in here! Make sure to take a look at the [changelog](https://codeberg.org/monocles/monocles_chat/src/branch/master/fastlane/metadata/android/en-US/changelogs/19301.txt) for all the details!
+- [Prose](https://prose.org/) has released versions [0.13.0](https://github.com/prose-im/prose-app-web/releases/tag/0.13.0) of its web frontend [prose-app-web](https://github.com/prose-im/prose-app-web).
+- [prose-core-client](https://github.com/prose-im/prose-core-client) has released versions [0.1.107](https://github.com/prose-im/prose-core-client/releases/tag/0.1.107), [0.1.108](https://github.com/prose-im/prose-core-client/releases/tag/0.1.108), [0.1.109](https://github.com/prose-im/prose-core-client/releases/tag/0.1.109) and [0.1.110](https://github.com/prose-im/prose-core-client/releases/tag/0.1.110) of its core XMPP client manager and protocols. You can read all the details on their [changelog](https://github.com/prose-im/prose-core-client/blob/master/CHANGELOG.md).
+- [Psi+](https://sourceforge.net/projects/psiplus/) has released version [1.5.2125.0 installer](https://sourceforge.net/projects/psiplus/files/Windows/Personal-Builds/KukuRuzo/) of its development branch of the [Psi](https://sourceforge.net/projects/psi/) XMPP client.
```

Comment:
I'm actually wondering if development branches are reaching the level of news already.